manifests/director/catalog.pp

```diff
@@ -129,7 +129,7 @@
 define bareos::director::catalog (
   $ensure = present,
   $db_address = undef,
-  $db_driver = undef,
+  Enum['postgresql'] $db_driver = 'postgresql',
```

Comment:
Not a user of the module, this looks fine and the only thing I am wondering is related to the new default value: what happened before if you did not explicitly set `db_driver`?  I understand it would fail, so everyone has db_driver set to something, and in this case it LGTM.

But if not setting a value was using something else that PostgreSQL, it would be better to not set a default value and force users to explicitly say they use PostgreSQL in their config (that way one cannot miss the fact that they are breaking their system by changing the database system used).

Reply:
As of 21, if `DB Driver` is deprecated, the default is `postgresql` and the only possible value is `postgresql`.

I've reworked the PR so that the `db_driver` parameter is labeled as deprecated but doesn't force setting the value to `postgresql`.